.github/workflows/deploy-binance.yml

```diff
@@ -0,0 +1,148 @@
+name: Deploy to Binance smart chain
+
+# Run on pushes to master or PRs to master
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  # Launches build when release is published
+  release:
+      types: [published]
+
+env:
+  REPO_NAME_ALPHANUMERIC: safereact
+  REACT_APP_NETWORK: 'binance'
+  STAGING_BUCKET_NAME: ${{ secrets.STAGING_BINANCE_BUCKET_NAME }}
+  REACT_APP_SENTRY_DSN: ${{ secrets.SENTRY_DSN_BINANCE }}
```

Comment:
Maybe let's use BSC for variable names? I've already added this secret on GitHub as `SENTRY_DSN_BSC`.
Same for the analytics and bucket keys.

Reply:
If the env vars are created like that there is no problem

Reply:
Do you think we should refire at all places like this? At first I thought about binance to be more explicit as bsc is a bit confusing if you don't know what you are talking about, and binance_smart_chain to be very long. Also there is a legacy network that is named binance chain, so maybe also being explicit about "Binance Smart Chain" instead the legacy one it's also something good

Reply:
I think it's OK to use BINANCE in other places. Easier to understand, like you said.
But let's call it BSC in the domains, buckets and other secrets.